packages/main/src/Tokenizer.ts

```diff
@@ -211,6 +214,26 @@ class Tokenizer extends UI5Element {
 	@property({ type: Boolean })
 	readonly = false;
 
+	/**
+	 * Defines whether tokens are displayed on multiple lines.
```

Comment:
I couldn't find anywhere in the project where this flag is used on property level. I believe that since the whole component is flagged as experimental, it's API is subject to changes.

Reply:
I am using for reference the following PR. Please check it -  https://github.com/SAP/ui5-webcomponents/pull/9304 and probably we could discuss it with the team

Reply:
As discussed, expertimental properties should have a note.